lib/SILOptimizer/Mandatory/TFLowerGraph.cpp

```diff
@@ -2011,15 +1941,16 @@ GLStatus TFGraphLowering::visitTFOpInst(BuiltinInst *inst) {
   SILTensorOpInfo tfopInfo = SILTensorOpInfo::decode(inst).getValue();
 
   // Swift host <-> TF device sends/recvs.
-  assert(tfopInfo.opName != "tfc.SendToHost"); // Already switched to graph op.
-  if (tfopInfo.opName == "tfc.RecvFromHost")
-    return visitBuiltinRecvFromHostInst(tfopInfo);
+  if (tfopInfo.opName == "tfc.SendToHost" ||
+      tfopInfo.opName == "tfc.RecvFromHost")
+    assert(0 &&
+           "Host sends/recvs are only supported in strict deabstraction mode!");
 
   // Device-to-device sends/recvs.
-  if (tfopInfo.opName == "tfc.D2DTensorRecv")
-    return visitBuiltinD2DTensorRecvInst(tfopInfo);
-  else if (tfopInfo.opName == "tfc.D2DTensorSend")
-    return visitBuiltinD2DTensorSendInst(tfopInfo);
+  if (tfopInfo.opName == "tfc.D2DTensorRecv" ||
+      tfopInfo.opName == "tfc.D2DTensorSend")
+    assert(0 &&
```

Comment:
Please put if conditions into the assert condition

Reply:
Done.